crates/nu-system/src/macos.rs

```diff
@@ -336,7 +344,7 @@ impl ProcessInfo {
         }
     }
 
-    /// Full name of comand only
+    /// Full name of command only
     pub fn command_only(&self) -> String {
```

Comment:
Let me know if you want me to remove this..

Reply:
if it's not being used, it may be good to clean things up and remove it.